src/Arrows.js

```diff
@@ -1,68 +1,105 @@
-import React, { Component, PropTypes } from 'react';
+import { default as React, Component, PropTypes } from 'react';
```

Comment:
This looks very odd to my eyes. It isn't the normal style people use. Can you revert this back? If you don't want to, I will revert it after merging this PR.